src/hyperion/device_setup_plans/dcm_pitch_roll_mirror_adjuster.py

```diff
@@ -0,0 +1,132 @@
+import json
+
+import bluesky.plan_stubs as bps
+from dodal.devices.DCM import DCM, fixed_offset_from_beamline_params
+from dodal.devices.focusing_mirror import (
+    FocusingMirror,
+    MirrorStripe,
+    VFMMirrorVoltages,
+)
+
+from hyperion.device_setup_plans.adjuster import LUTAdjuster
+from hyperion.log import LOGGER
+from hyperion.parameters.beamline_parameters import (
+    GDABeamlineParameters,
+)
+from hyperion.utils.lookup_table import (
+    LinearInterpolationLUTConverter,
+)
+
+MIRROR_VOLTAGE_GROUP = "MIRROR_VOLTAGE_GROUP"
+DCM_GROUP = "DCM_GROUP"
+
+
+def _apply_and_wait_for_voltages_to_settle(
+    stripe: MirrorStripe, mirror: FocusingMirror, mirror_voltages: VFMMirrorVoltages
+):
+    with open(mirror_voltages.voltage_lookup_table_path) as lut_file:
+        json_obj = json.load(lut_file)
+
+    # sample mode is the only mode supported
+    sample_data = json_obj["sample"]
+    mirror_key = mirror.name.lower()
+    if stripe == MirrorStripe.BARE:
+        stripe_key = "bare"
+    elif stripe == MirrorStripe.RHODIUM:
+        stripe_key = "rh"
+    elif stripe == MirrorStripe.PLATINUM:
+        stripe_key = "pt"
+    else:
+        raise ValueError(f"Unsupported stripe '{stripe}'")
+
+    required_voltages = sample_data[stripe_key][mirror_key]
+    for i in range(0, len(required_voltages)):
+        voltage_channel = mirror_voltages.voltage_channels[i]
+        LOGGER.debug(
+            f"Applying and waiting for voltage {voltage_channel.name} = {required_voltages[i]}"
+        )
+        yield from bps.abs_set(
+            voltage_channel, required_voltages[i], group=MIRROR_VOLTAGE_GROUP
+        )
+
+    yield from bps.wait(group=MIRROR_VOLTAGE_GROUP)
+
+
+def adjust_mirror_stripe(
+    energy_kev, mirror: FocusingMirror, mirror_voltages: VFMMirrorVoltages
+):
+    # Transmission should be 100% and feedback should be OFF prior to entry
```

Comment:
have changed it to a docstring. What I really wanted to do was put some kind of assert here but there didn't seem to be an easy way to do so without being overly heavyweight.
Agree about transmission, I mainly included it because all existing GDA code does this, and so does our existing feedback decorator.